recipes/flask_swagger_ui/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "flask-swagger-ui" %}
+{% set version = "3.6.0" %}
+{% set sha256 = "afe84e4721b9b4bdd3efc2f9949a9f990f8e63507aedcf450d00856cf669f005" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - flask
+  run:
+    - python
+    - flask
+
+test:
+  imports:
+    - flask_swagger_ui
+
+about:
+  home: https://github.com/sveint/flask-swagger-ui
+  license: MIT
+  summary: 'Swagger UI blueprint for Flask'
+  license_family: MIT
```

Comment:
Is there a licence file associated with this?

Reply:
There is, but it does not have an extension and it throws an exception with conda build.

Reply:
That's odd, most licences I've had in recipes don't have extensions.

Reply:
Do you have the exception?

Reply:
When I add the line 
```yaml
license_file: LICENSE
```
I get this after some

```bash
$ conda build flask_swagger_ui                                                                                               
BUILD START: flask-swagger-ui-3.6.0-py_0
    (actual version deferred until further download or env creation)

The following NEW packages will be INSTALLED:

    ca-certificates: 2017.11.5-0      conda-forge
    certifi:         2017.11.5-py36_0 conda-forge
    click:           6.7-py36_0       conda-forge
    flask:           0.12.2-py36_0    conda-forge
    itsdangerous:    0.24-py_2        conda-forge
    jinja2:          2.10-py36_0      conda-forge
    markupsafe:      1.0-py36_0       conda-forge
    ncurses:         5.9-10           conda-forge
    openssl:         1.0.2m-0         conda-forge
    pip:             9.0.1-py36_0     conda-forge
    python:          3.6.3-1          conda-forge
    readline:        6.2-0            conda-forge
    setuptools:      38.2.3-py36_0    conda-forge
    sqlite:          3.13.0-1         conda-forge
    tk:              8.5.19-2         conda-forge
    werkzeug:        0.12.2-py_1      conda-forge
    wheel:           0.30.0-py_1      conda-forge
    xz:              5.2.3-0          conda-forge
    zlib:            1.2.11-0         conda-forge


WARNING: conda-build appears to be out of date. You have version 2.1.18 but the
latest version is 3.1.0. Run

conda update -n root conda-build

to get the latest version.

Source cache directory is: /Users/ukaratay/miniconda3/conda-bld/src_cache
Found source in cache: flask-swagger-ui-3.6.0.tar.gz
Extracting download
BUILD START (revised): flask-swagger-ui-3.6.0-py_0
Package: flask-swagger-ui-3.6.0-py_0
source tree in: /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/work/flask-swagger-ui-3.6.0
+ source /Users/ukaratay/miniconda3/bin/activate /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_
+ python setup.py install --single-version-externally-managed --record record.txt
running install
running build
running build_py
creating build
creating build/lib
creating build/lib/flask_swagger_ui
copying flask_swagger_ui/flask_swagger_ui.py -> build/lib/flask_swagger_ui
copying flask_swagger_ui/__init__.py -> build/lib/flask_swagger_ui
copying flask_swagger_ui/README.md -> build/lib/flask_swagger_ui
creating build/lib/flask_swagger_ui/templates
copying flask_swagger_ui/templates/index.template.html -> build/lib/flask_swagger_ui/templates
creating build/lib/flask_swagger_ui/dist
copying flask_swagger_ui/dist/VERSION -> build/lib/flask_swagger_ui/dist
copying flask_swagger_ui/dist/LICENSE -> build/lib/flask_swagger_ui/dist
copying flask_swagger_ui/dist/oauth2-redirect.html -> build/lib/flask_swagger_ui/dist
copying flask_swagger_ui/dist/swagger-ui.js -> build/lib/flask_swagger_ui/dist
copying flask_swagger_ui/dist/swagger-ui-bundle.js -> build/lib/flask_swagger_ui/dist
copying flask_swagger_ui/dist/swagger-ui-standalone-preset.js -> build/lib/flask_swagger_ui/dist
copying flask_swagger_ui/dist/swagger-ui.css -> build/lib/flask_swagger_ui/dist
copying flask_swagger_ui/dist/favicon-16x16.png -> build/lib/flask_swagger_ui/dist
copying flask_swagger_ui/dist/favicon-32x32.png -> build/lib/flask_swagger_ui/dist
running install_lib
creating /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui
creating /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui/dist
copying build/lib/flask_swagger_ui/dist/favicon-16x16.png -> /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui/dist
copying build/lib/flask_swagger_ui/dist/LICENSE -> /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui/dist
copying build/lib/flask_swagger_ui/dist/swagger-ui.js -> /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui/dist
copying build/lib/flask_swagger_ui/dist/swagger-ui.css -> /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui/dist
copying build/lib/flask_swagger_ui/dist/swagger-ui-bundle.js -> /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui/dist
copying build/lib/flask_swagger_ui/dist/swagger-ui-standalone-preset.js -> /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui/dist
copying build/lib/flask_swagger_ui/dist/VERSION -> /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui/dist
copying build/lib/flask_swagger_ui/dist/oauth2-redirect.html -> /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui/dist
copying build/lib/flask_swagger_ui/dist/favicon-32x32.png -> /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui/dist
copying build/lib/flask_swagger_ui/flask_swagger_ui.py -> /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui
copying build/lib/flask_swagger_ui/__init__.py -> /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui
copying build/lib/flask_swagger_ui/README.md -> /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui
creating /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui/templates
copying build/lib/flask_swagger_ui/templates/index.template.html -> /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui/templates
byte-compiling /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui/flask_swagger_ui.py to flask_swagger_ui.cpython-36.pyc
byte-compiling /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui/__init__.py to __init__.cpython-36.pyc
running install_egg_info
running egg_info
writing flask_swagger_ui.egg-info/PKG-INFO
writing dependency_links to flask_swagger_ui.egg-info/dependency_links.txt
writing requirements to flask_swagger_ui.egg-info/requires.txt
writing top-level names to flask_swagger_ui.egg-info/top_level.txt
reading manifest file 'flask_swagger_ui.egg-info/SOURCES.txt'
writing manifest file 'flask_swagger_ui.egg-info/SOURCES.txt'
Copying flask_swagger_ui.egg-info to /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.6/site-packages/flask_swagger_ui-3.6.0-py3.6.egg-info
running install_scripts
writing list of installed files to 'record.txt'
INFO:conda_build.build:Packaging flask-swagger-ui-3.6.0-py_0
number of files: 19
Fixing permissions
Fixing permissions
Traceback (most recent call last):
  File "/Users/ukaratay/miniconda3/lib/python3.6/site-packages/conda_build/utils.py", line 133, in _copy_with_shell_fallback
    stderr=subprocess.PIPE, stdout=subprocess.PIPE)
  File "/Users/ukaratay/miniconda3/lib/python3.6/subprocess.py", line 291, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'cp -a /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/work/flask-swagger-ui-3.6.0/LICENSE /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/info/LICENSE.txt' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ukaratay/miniconda3/bin/conda-build", line 6, in <module>
    sys.exit(conda_build.cli.main_build.main())
  File "/Users/ukaratay/miniconda3/lib/python3.6/site-packages/conda_build/cli/main_build.py", line 342, in main
    execute(sys.argv[1:])
  File "/Users/ukaratay/miniconda3/lib/python3.6/site-packages/conda_build/cli/main_build.py", line 333, in execute
    noverify=args.no_verify)
  File "/Users/ukaratay/miniconda3/lib/python3.6/site-packages/conda_build/api.py", line 97, in build
    need_source_download=need_source_download, config=config)
  File "/Users/ukaratay/miniconda3/lib/python3.6/site-packages/conda_build/build.py", line 1524, in build_tree
    config=config)
  File "/Users/ukaratay/miniconda3/lib/python3.6/site-packages/conda_build/build.py", line 1159, in build
    built_package = bundlers[output_dict.get('type', 'conda')](output_dict, m, config, env)
  File "/Users/ukaratay/miniconda3/lib/python3.6/site-packages/conda_build/build.py", line 896, in bundle_conda
    create_info_files(metadata, files, config=config, prefix=config.build_prefix)
  File "/Users/ukaratay/miniconda3/lib/python3.6/site-packages/conda_build/build.py", line 494, in create_info_files
    copy_license(m, config)
  File "/Users/ukaratay/miniconda3/lib/python3.6/site-packages/conda_build/build.py", line 272, in copy_license
    locking=config.locking)
  File "/Users/ukaratay/miniconda3/lib/python3.6/site-packages/conda_build/utils.py", line 177, in copy_into
    _copy_with_shell_fallback(src, dst_fn)
  File "/Users/ukaratay/miniconda3/lib/python3.6/site-packages/conda_build/utils.py", line 136, in _copy_with_shell_fallback
    raise OSError("Failed to copy {} to {}.  Error was: {}".format(src, dst, e))
OSError: Failed to copy /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/work/flask-swagger-ui-3.6.0/LICENSE to /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/info/LICENSE.txt.  Error was: Command 'cp -a /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/work/flask-swagger-ui-3.6.0/LICENSE /Users/ukaratay/miniconda3/conda-bld/flask-swagger-ui_1512513987100/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/info/LICENSE.txt' returned non-zero exit status 1.
Exception ignored in: <bound method BaseFileLock.__del__ of <filelock.UnixFileLock object at 0x11840ed68>>
Traceback (most recent call last):
  File "/Users/ukaratay/miniconda3/lib/python3.6/site-packages/filelock.py", line 305, in __del__
  File "/Users/ukaratay/miniconda3/lib/python3.6/site-packages/filelock.py", line 292, in release
  File "/Users/ukaratay/miniconda3/lib/python3.6/site-packages/filelock.py", line 371, in _release
AttributeError: 'NoneType' object has no attribute 'flock'
```

Reply:
The error is because the License was not packaged with the code on pypi (it doesn't exist in the tar.gz). Could you please open up an issue at the source code asking for the license to be included in the pypi package? Also can you please put a comment into this recipe with the issue URL?

Reply:
Here it is: sveint/flask-swagger-ui#17. Do you want me to add it to the recipe file as well?

Reply:
Yes but if wait until the osx build passes then add it and skip the CI. That way you don't need to wait for it it build again.

Reply:
@CJ-Wright Added the comment to the recipe. Can you skip the CI? I don't think I have the permission.

Reply:
The way you skip the CI is putting [ci skip] in the commit message.

Reply:
Didn't know that one :)

Reply:
I don't know if we have docs up for that yet (I'm checking now)

Reply:
@CJ-Wright can you merge this now?

Reply:
Sure, when the licence is released please create a PR against the feedstock to include it and bump the `build_number`.

Reply:
Will do.